<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepChose</class>
 <widget class="QDialog" name="RepChose">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выберите необходимый репозиторй</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Доступные репертории</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Уровни</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RepChose</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RepChose</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RepChose</receiver>
   <slot>finding(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>RepChose</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>RepChose</receiver>
   <slot>showLevels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>finding(QString)</slot>
  <slot>showLevels()</slot>
 </slots>
</ui>
